README.md

```diff
@@ -1,2 +1,25 @@
 # to-do-list
 In this project, this simple To-do-list web App. was built using webpack and served by a webpack dev server.
+
+## Built with 
+    Major Languages
+    - Html and CSS
+    - Javascript
+    
+    Major Technologies
+    - VS code as IDE
+    - Git and Github
+    - Linters
+
+## setup details 
```

Comment:
- [ ] [optional] It would be nice to include how to clone and run the project locally as part of your project documentation, it will make the readme file more attractive and professional.